Source/WebKit/Platform/cocoa/ImageAnalysisUtilities.h

```diff
@@ -61,14 +61,35 @@ RetainPtr<CocoaImageAnalyzerRequest> createImageAnalyzerRequest(CGImageRef, VKAn
 void requestVisualTranslation(CocoaImageAnalyzer *, NSURL *, const String& source, const String& target, CGImageRef, CompletionHandler<void(WebCore::TextRecognitionResult&&)>&&);
 void requestBackgroundRemoval(CGImageRef, CompletionHandler<void(CGImageRef)>&&);
 
+constexpr VKAnalysisTypes analysisTypesForFullscreenVideo()
```

Comment:
Would it make sense for this to call `analysisTypesForElementFullscreenVideo`, or do we expect the two cases to diverge further?

Reply:
That's a good point — I think we can share (most of) this code!